src/content/guidelines/accessibility/keyboard.md

```diff
@@ -1,28 +1,21 @@
 ## Keyboard accessibility
-Keyboard accessibility is important for users who rely on or prefer using a keyboard. Encourage accessibility by providing keyboard functionality to all available content. Common keyboard interactions include using the `Tab` key to select different interactive elements on a page and using the `Enter` key or the Spacebar to activate an in-focus element.
+Keyboard accessibility is required and is helpful for users who rely on or prefer using a keyboard. Make sure to provide keyboard functionality to all interactive content. Common keyboard interactions include using the `Tab` key to select different interactive elements on a page and using the `Enter` key or the `Spacebar` to activate an in-focus element.
```

Comment:
Are there systematic patterns with arrow keys that we could include here, as well? Or is the usage too different compared to tab or enter/spacebar.

Reply:
I think this section needs to be built out a bit more. This is okay for now but isn't super helpful.

Reply:
I'll add this as an issue locally and fix that section for future release.